sky/clouds/gcp.py

```diff
@@ -470,6 +470,24 @@ def get_vcpus_mem_from_instance_type(
         return service_catalog.get_vcpus_mem_from_instance_type(instance_type,
                                                                 clouds='gcp')
 
+    @classmethod
+    def _find_application_key_path(cls) -> str:
+        # Check the application default credentials in the environment variable.
+        # If the file does not exist, fallback to the default path.
+        application_key_path = os.environ.get(_GCP_APPLICATION_CREDENTIAL_ENV,
+                                              None)
+        if application_key_path:
```

Comment:
```suggestion
        if application_key_path is not None:
```
nit: styleguide

Reply:
This is to check both the empty string and the `None`. The original `if application_key_path:` should be fine? Will add a comment to it.